buying_selling/users/models.py

```diff
@@ -106,6 +114,7 @@ class Profile(models.Model):
         ('BE-MBA(ECE)', 'Electroincs MBA Dual Degree'),
         ('Others', 'Other'),
     )
+    hostel = models.ManyToManyField(Hostel)
```

Comment:
Why a `ManyToMany` Field?

Reply:
I guess at any point of time we should only keep one hostel linked with a profile

Reply:
Everything else is prefect!

Reply:
yes you are right, I made a mistake, it should be foreign key, I've updated it in my new commit